query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/NoErrorOnEmptyNestedDisconnectSpec.scala

```diff
@@ -0,0 +1,65 @@
+package writes.nestedMutations
```

Comment:
Amendment wrt. the other PR: The same applies here - no legacy queries anymore. This is not really a regression spec so the other point doesn't apply, but I still believe the tests should be merged into existing ones, because the amount of free-flying tests is getting too much for me to handle efficiently.

Reply:
Added a test case to an existing test